a1111/webui-user.sh

```diff
@@ -10,7 +10,7 @@ install_dir="/workspace"
 #clone_dir="stable-diffusion-webui"
 
 # Commandline arguments for webui.py, for example: export COMMANDLINE_ARGS="--medvram --opt-split-attention"
-export COMMANDLINE_ARGS="--port 3001 --skip-install --listen --api --xformers --enable-insecure-extension-access --no-half-vae"
+test -z "$COMMANDLINE_ARGS" || export COMMANDLINE_ARGS="--port 3001 --skip-install --listen --api --xformers --enable-insecure-extension-access --no-half-vae"
```

Comment:
This does not work correctly, correct syntax is:
```bash
: ${COMMANDLINE_ARGS:="--port 3001 --skip-install --listen --api --xformers --enable-insecure-extension-access --no-half-vae"} && export COMMANDLINE_ARGS
```

Reply:
Out of a similar need I've decided to add something along the lines of :

```sh
WEBUI_USER_LOCAL=${WEBUI_USER_LOCAL:- webui-user.local.sh}
if [ -x "$WEBUI_USER_LOCAL" ] ; then
  source "$WEBUI_USER_LOCAL"
if
```